tensorboard/plugins/graph/tf_graph_common/edge.ts

```diff
@@ -422,8 +420,9 @@ function position(component: HTMLElement, edgeGroup: HTMLElement) {
   d3.select(edgeGroup)
       .select('path.' + Class.Edge.LINE)
       .transition()
-      .attrTween('d', (d: EdgeData, i: number, a: SVGPathElement[]) => {
-        return getEdgePathInterpolator(component, d, i, a);
+      .attrTween('d', function(d: EdgeData, i: number, a: SVGPathElement[]) {
+        return getEdgePathInterpolator(
+            component, <SVGPathElement>this, d, i, a);
```

Comment:
Prefer `(this as SVGPathElement)`, as per ts-style and to not conflict
with JSX.